tests/firewall/firewall-06-tls-sni-enforce/test.yaml

```diff
@@ -51,18 +47,6 @@ checks:
     count: 26
     match:
       event_type: drop
-- filter:
-    min-version: 8
-    count: 0
-    match:
-      event_type: alert
-      pkt_src: "stream (flow timeout)"
```

Comment:
no value in keeping this check?

Reply:
It is improved by checking the alerts are all wire/pcap